libs/SalesforceSDKCore/SalesforceSDKCore/UIColor+SFColorsTests.m

```diff
@@ -0,0 +1,64 @@
+//
```

Comment:
@rafael-chavez Change copyright to our standard copyright. Move this file under `SalesforceSDKCoreTests` folder.